### ISSUE TITLE
some commands do nothing on a tekkit server

### ISSUE THREAD

**Issue by diehardbarrett** [GriefPrevention/GriefPrevention#531]
New Tekkit classic 3.1.2 server; Everything seems normal about this plugin accept /buyclaimblocks and /sellclaimblocks both do nothing when typed. No error in console, nothing happes If a block amount is entered as well. Op/admin/default all have the same problem. If given claim blocks, everything works fine.

**Comment by diehardbarrett** [comment]
delete

**Comment by RoboMWM** [comment]
Please do not try to delete a post/issue if the issue has been resolved. Instead, post about how you fixed the issue. It may help others who cone across the same issue and are searching for answers in the future.

I'll unlock once I get on a desktop to restore the original issue text.

**Comment by RoboMWM** [comment]
OP must've realized that tekkit refers to an ancient version of Minecraft.

**Comment by diehardbarrett** [comment]
Well yes, but actually no. I recently setup a tekkit classic server (minecraft 1.2.5) and was having an issue where 2 commands (/buyclaimblocks and /sellclaimblocks) of the grief prevention plugin weren't working. They wouldn't generate an error in console, or give the generic "unknown command" message to the user. the commands literally did nothing. I read deep in the Jar file that "Vault" must be installed, but many versions of vault didn't work. But after removing, reinstalling both vault and essentials (which was blocking vault from taking over permission controls) many times, in various orders, it was fixed. I added essentials (Dev2.9.337) and then added Vault (1.2.16). Thats what my fix was.
